addons/hr_holidays/models/hr_leave.py

```diff
@@ -572,6 +577,11 @@ def _read_from_database(self, field_names, inherited_field_names=[]):
 
     @api.multi
     def write(self, values):
+        # Allow an employee to always write his own out of office message
+        if len(self) == 1 and values.keys() == {'out_of_office_message'} and self.employee_id.user_id == self.env.user:
+            values = {'out_of_office_message': values.get('out_of_office_message')}
```

Comment:
I believe that this line is not really useful as you can only get here if you write only `out_of_office_message`, hence this line is just copying `values`.

Reply:
True. Fixed